libs/auth/scripts/src/configure_java_card.ts

```diff
@@ -262,25 +270,26 @@ async function runAppletConfigurationCommands(): Promise<void> {
   }
 }
 
-async function createAndStoreCardVxCert(): Promise<void> {
+async function createAndStoreCardVxCert({
+  javaCardConfig,
+}: ScriptEnv): Promise<void> {
   sectionLog('🔏', 'Creating and storing card VotingWorks cert...');
-  const card = new JavaCard({ vxCertAuthorityCertPath });
+
+  const card = new JavaCard(javaCardConfig);
   await waitForReadyCardStatus(card);
-  await card.createAndStoreCardVxCert({
-    source: 'file',
-    path: vxPrivateKeyPath,
-  });
+  await card.createAndStoreCardVxCert();
 }
 
 /**
- * An initial Java Card configuration script to be run at the Bakery
+ * An initial Java Card configuration script to be run at a VotingWorks facility
  */
 export async function main(): Promise<void> {
   try {
     checkForScriptDependencies();
+    const scriptEnv = readScriptEnvVars();
```

Comment:
Moved env var reading from outside `main` into `main` in all relevant scripts. One positive side effect of this is cleaner error messages for missing env vars.

_Before_

```
$ ./scripts/configure-java-card 

/home/vx/code/vxsuite/libs/basics/build/assert.js:10
        throw new Error(message);
              ^
Error: Missing required NODE_ENV env var
    at assert (/home/vx/code/vxsuite/libs/basics/build/assert.js:10:15)
    at getRequiredEnvVar (/home/vx/code/vxsuite/libs/auth/src/env_vars.ts:10:3)
    at getNodeEnv (/home/vx/code/vxsuite/libs/auth/src/env_vars.ts:19:19)
    at isNodeEnvProduction (/home/vx/code/vxsuite/libs/auth/src/env_vars.ts:31:10)
    at shouldUseProdCerts (/home/vx/code/vxsuite/libs/auth/src/config.ts:25:10)
    at constructJavaCardConfigForVxProgramming (/home/vx/code/vxsuite/libs/auth/src/config.ts:119:28)
    at Object.<anonymous> (/home/vx/code/vxsuite/libs/auth/scripts/src/configure_java_card.ts:36:11)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Module._compile (/home/vx/code/vxsuite/node_modules/.pnpm/source-map-support@0.5.21/node_modules/source-map-support/source-map-support.js:568:25)
    at Module.mod._compile (/home/vx/code/vxsuite/node_modules/.pnpm/esbuild-runner@2.2.2_esbuild@0.17.11/node_modules/esbuild-runner/src/hook.ts:53:22)
```

_After_

```
$ ./scripts/configure-java-card 
❌ Missing required NODE_ENV env var
```